delegation-toolkit/how-to/redeem-delegation.md

```diff
@@ -207,10 +208,12 @@ the primary difference is that an EOA submits a regular transaction instead of a
 
 ```typescript
 import { 
-  DelegationFramework, 
-  SINGLE_DEFAULT_MODE, 
-  ExecutionStruct
+  ExecutionStruct,
+  Delegation,
+  ExecutionMode,
 } from "@metamask/delegation-toolkit";
+import { DelegationManager } from "@metamask/delegation-toolkit/contracts";
+import { SINGLE_DEFAULT_MODE } from "@metamask/delegation-toolkit/utils";
```

Comment:
Not for docs, but seems trange to me that ExecutionMode is in `@metamask/delegation-toolkit` but the actual modes are in `@metamask/delegation-toolkit/utils`.

I think we should combine these into an enum `enum ExecutionMode` - I'll pick that up.